extensions/stackdriver/stackdriver.cc

```diff
@@ -211,6 +211,7 @@ void StackdriverRootContext::record() {
   ::Extensions::Stackdriver::Metric::record(isOutbound(), local_node_info_,
                                             peer_node_info, request_info);
   if (enableServerAccessLog() && shouldLogThisRequest()) {
+    ::Wasm::Common::populateExtendedHTTPRequestInfo(&request_info);
```

Comment:
Can this be called for TCP?

Reply:
Most information populated in this function are http based, except source/dst addresses. If it is desired, we could add those two fields into populateTCPRequestInfo.